spec/models/estimated_population_spec.rb

```diff
@@ -0,0 +1,121 @@
+require "rails_helper"
+
+RSpec.describe EstimatedPopulation, type: :model do
+  describe "validations" do
+    it "is not valid without a population" do
+      region = Region.create!(name: "State", region_type: "state", reparent_to: Region.root)
+      estimated_population = EstimatedPopulation.new(diagnosis: "HTN", region_id: region.id)
+
+      expect(estimated_population).to_not be_valid
```

Comment:
Can you make these validity assertions more specific? For example, for this one:

```
expect(estimated_population.errors[:population]).to eq (["must exist"])
```

The others could verify their specific validations as well.